bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/HistoricalPersistenceStoreService.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+/**
+ * This class provides an interface to the a {@link PersistenceService} to allow data to be stored
+ * at a specific time. This allows bindings that interface to devices that store data internally,
+ * and then periodically provide it to the server to be accommodated.
+ *
+ * @author Chris Jackson - Initial implementation and API
+ *
+ */
+public interface HistoricalPersistenceStoreService {
```

Comment:
> Right, but if we combine it with your other "separate interface to come", it will be :-)

Yes, if we combine into a single interface, then this is fine with me.